lib/coophub_web/templates/layout/app.html.eex

```diff
@@ -19,6 +19,12 @@
     <meta name="msapplication-config" content="<%= Routes.static_path(@conn, "/icons/browserconfig.xml") %>">
     <meta name="theme-color" content="#ffffff">
 
+    <meta property="og:title" content="CoopHub · Open source from co-operatives" />
```

Comment:
I would add "worldwide" to: Open source from worldwide co-operatives
But it's just a silly suggestion.